lib/recurly/webhook.rb

```diff
@@ -37,6 +37,7 @@ module Webhook
     autoload :SubscriptionResumedNotification,      'recurly/webhook/subscription_resumed_notification'
     autoload :CanceledSubscriptionNotification,     'recurly/webhook/canceled_subscription_notification'
     autoload :ScheduledSubscriptionPauseNotification,'recurly/webhook/scheduled_subscription_pause_notification'
+    autoload :ScheduledSubscriptionUpdate,          'recurly/webhook/scheduled_subscription_update_notification'
```

Comment:
Needs to be `autoload :ScheduledSubscriptionUpdateNotification`

Reply:
Tests should pass after that's fixed

Reply:
Would you believe that I actually have this fixed in an open buffer, but had not saved the file in two days?

Reply:
Would you believe that I actually have this fixed in an open buffer, but had not saved the file in two days?